lib/views/commit-view.js

```diff
@@ -196,8 +219,8 @@ export default class CommitView extends React.Component {
     /* eslint-disable max-len */
     const svgPath = 'M9.875 2.125H12v1.75H9.875V6h-1.75V3.875H6v-1.75h2.125V0h1.75v2.125zM6 6.5a.5.5 0 0 1-.5.5h-5a.5.5 0 0 1-.5-.5V6c0-1.316 2-2 2-2s.114-.204 0-.5c-.42-.31-.472-.795-.5-2C1.587.293 2.434 0 3 0s1.413.293 1.5 1.5c-.028 1.205-.08 1.69-.5 2-.114.295 0 .5 0 .5s2 .684 2 2v.5z';
     return (
-      <svg className="github-CommitView-coAuthorToggleIcon" viewBox="0 0 12 7" xmlns="http://www.w3.org/2000/svg">
-        <title>Toggle co-authors</title>
+      <svg className={cx('github-CommitView-coAuthorToggleIcon', {focused: this.state.showCoAuthorInput})} viewBox="0 0 12 7" xmlns="http://www.w3.org/2000/svg">
+        <title>Toggle Add or remove co-authors</title>
```

Comment:
🚨 🎨  Double capital here: `Toggle Add [...]`

The wording is also a bit awkward here. How about something like "Toggle co-author entry"? @simurai might have ideas too :smile:

Reply:
I meant to change this from "Toggle co author" to "Add/remove co authors" but I fat fingered it. Oops! Will fix. And yeah, I agree the wording could be better.